src/pages/view-test-result/cat-b/components/test-summary-card/test-summary-card.html

```diff
@@ -4,91 +4,35 @@ <h4>Test summary</h4>
-      </ion-row>
-      <ion-row class="mes-data-row-with-separator">
-        <ion-col col-40>
-          <label>Route number</label>
-        </ion-col>
-        <ion-col col-56>
-          <span class="mes-data" *ngIf="!data.routeNumber">None</span>
-          <span class="mes-data">{{ data.routeNumber }}</span>
-        </ion-col>
-      </ion-row>
-      <ion-row class="mes-data-row-with-separator">
-        <ion-col col-40>
-          <label>Independent driving</label>
-        </ion-col>
-        <ion-col col-56>
-            <span class="mes-data" *ngIf="!data.independentDriving">None</span>
-          <span class="mes-data">{{ data.independentDriving }}</span>
-        </ion-col>
-      </ion-row>
-      <ion-row class="mes-data-row-with-separator">
-        <ion-col col-40>
-          <label>Physical description of candidate</label>
-        </ion-col>
-        <ion-col col-56>
-          <span class="mes-data" *ngIf="!data.candidateDescription">None</span>
-          <span class="mes-data">{{ data.candidateDescription }}</span>
-        </ion-col>
-      </ion-row>
-      <ion-row class="mes-data-row-with-separator">
-        <ion-col col-40>
-          <label>Debrief witnessed</label>
-        </ion-col>
-        <ion-col col-56>
-          <span class="mes-data">{{ getConvertBooleanToString(data.debriefWitnessed) }}</span>
-        </ion-col>
-      </ion-row>
-      <ion-row class="mes-data-row-with-separator">
-        <ion-col col-40>
-          <label>Weather conditions</label>
-        </ion-col>
-        <ion-col col-56>
-          <span class="mes-data" *ngIf="data.weatherConditions.length === 0">None</span>
-          <span class="mes-data">{{ getFlattenArray(data.weatherConditions) }}</span>
-        </ion-col>
-      </ion-row>
-      <ion-row class="mes-data-row-with-separator">
-        <ion-col col-40>
-          <label>D255</label>
-        </ion-col>
-        <ion-col col-56>
-          <span class="mes-data">{{ getConvertBooleanToString(data.D255) }}</span>
-        </ion-col>
-      </ion-row>
-      <ion-row class="mes-data-row">
-        <ion-col col-40>
-          <label>Additional information</label>
-        </ion-col>
-        <ion-col col-56>
-          <span class="mes-data">{{ data.additionalInformation }}</span>
-        </ion-col>
-      </ion-row>
+      </data-row-custom>
+      <data-row
+        [label]="'Licence provided'"
+        [value]="getConvertBooleanToString(data.provisionalLicenceProvided)"
+        *ngIf="shouldDisplayLicenceProvided(data.provisionalLicenceProvided)">
+      </data-row>
+      <data-row [label]="'Certificate number'" [value]="data.passCertificateNumber" *ngIf="data.passCertificateNumber"></data-row>
+      <data-row-custom [label]="'Route number'">
+        <span class="mes-data" *ngIf="!data.routeNumber">None</span>
+        <span class="mes-data">{{ data.routeNumber }}</span>
+      </data-row-custom>
+      <data-row-custom [label]="'Independent driving'">
```

Comment:
Lots of these strings, could we perhaps use existing enums or create new ones?

Reply:
As they are only labels happy to do this as a new PR - we could also do it across the app :)